test/renate/benchmark/test_experimentation_config.py

```diff
@@ -34,6 +34,7 @@
 from renate.models.prediction_strategies import ICaRLClassificationStrategy
 
 
+@pytest.mark.slow
```

Comment:
how slow is this?

Reply:
Instantiating any of the transformer variants requires a download of the model which is slow.